AirCasting/Utils/ArrayUtils.swift

```diff
@@ -0,0 +1,33 @@
+// Created by Lunar on 14/06/2021.
+//
+
+import Foundation
+
+extension Array {
+    static var empty: Self { [] }
+}
+
+extension Array {
+    init(creating: @autoclosure () -> Element, times: Int) {
+        self = (0..<times).map { _ in creating() }
+    }
+}
+
+infix operator ~~
+
+extension Array where Element: Equatable {
+    static func ~~(lhs: Self, rhs: Self) -> Bool {
```

Comment:
NO!

Reply:
Well, I think that's quite handy in the unit test target and a "no" would not be sufficient to convince me, but I have a better idea: instead of using this let's create a custom assertion for that. I'll be happy and you'll be less mad 🐻

```Swift
assertContainsSameElements(removedSessions, sessionsToDelete)
```